influxdb3_write/src/catalog.rs

```diff
@@ -415,6 +446,12 @@ impl From<LastCacheSize> for usize {
     }
 }
 
+impl From<LastCacheSize> for u64 {
+    fn from(value: LastCacheSize) -> Self {
+        value.0 as u64
```

Comment:
I feel like using a `usize` for this is awkward given that the max size is 10, maybe using a smaller unsigned int directly will clean this up a bit.